core/src/main/java/io/grpc/ServerProvider.java

```diff
@@ -39,7 +39,6 @@ static final ServerProvider load(ClassLoader cl) {
 
     for (ServerProvider current : providers) {
       if (!current.isAvailable()) {
-        continue;
```

Comment:
The continue is more clear to me, and it is weird to have the block with no contents. It's honestly fine either way, so I think we should just leave it the way that it is.